### PR TITLE
Bug 1925083: IDP Creation not fully i18ned

### PR DESCRIPTION
**Pull request by rebeccaalpert**
"Add more," "More options," and "Browse" were not internationalized in the IDP Creation forms.

Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1925083.

### CODE REVIEW

File: frontend/public/components/utils/list-input.tsx [hunk 1]
```diff
@@ -3,8 +3,11 @@ import * as _ from 'lodash-es';
 import * as classNames from 'classnames';
 import { Button } from '@patternfly/react-core';
 import { PlusCircleIcon, MinusCircleIcon } from '@patternfly/react-icons';
+/* eslint-disable import/named */
```
Comment (spadgett) [inline frontend/public/components/utils/list-input.tsx:6]: Use disable next line instead

File: frontend/public/components/utils/file-input.tsx [hunk 2]
```diff
@@ -228,4 +233,5 @@ export type FileInputProps = {
   textareaFieldHelpText: string;
   isRequired: boolean;
   hideContents?: boolean;
+  t: TFunction;
```
Comment (spadgett) [inline frontend/public/components/utils/file-input.tsx:236]: Can you use the WithTranslation type instead?

File: frontend/public/components/utils/file-input.tsx [hunk 3]
```diff
@@ -81,7 +84,7 @@ export class FileInput extends React.Component<FileInputProps, FileInputState> {
               />
               <span className="pf-c-button pf-m-tertiary co-btn-file">
                 <input type="file" onChange={this.onFileUpload} />
-                Browse&hellip;
+                {t('public~Browse')}&hellip;
```
Comment (spadgett) [inline frontend/public/components/utils/file-input.tsx:87]: It’s probably better to make the ellipsis part of the message

**Comment by rebeccaalpert** [comment]
Feedback all set!

**Review by spadgett — APPROVED** [review]
/lgtm

**Pull request merged**